lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -14840,6 +14840,316 @@ msgstr "Eléments verrouillés"
+msgid "The configuration file %1 does not exists"
+msgstr "Le fichier de configuration %1 n'éxiste pas"
+
+#:Centreon/Domain/Downtime/DowntimeService.php:233
+msgid "Downtime not found"
+msgstr "Plage de maintenance non trouvée"
+
+#:Centreon/Domain/Downtime/DowntimeService.php:239
+msgid "Downtime already cancelled for this %1"
+msgstr "Plage de maintenance déjà annulée pour %1"
+
+#:Centreon/Domain/Downtime/DowntimeService.php:267
+msgid "Incorrect Resource Type"
+msgstr "Type de ressource incorrect"
+
+#:Centreon/Domain/Engine/EngineService.php:76
+#:Centreon/Domain/Engine/EngineService.php:102
+#:Centreon/Domain/Engine/EngineService.php:169
+#:Centreon/Domain/Engine/EngineService.php:220
+msgid "The contact alias is empty"
+msgstr "Le \"contact alias\" est vide"
+
+#:Centreon/Domain/Engine/EngineService.php:79
+#:Centreon/Domain/Engine/EngineService.php:129
+#:Centreon/Domain/Engine/EngineService.php:175
+#:Centreon/Domain/Engine/EngineService.php:266
+#:Centreon/Domain/Engine/EngineService.php:316
+#:Centreon/Domain/HostConfiguration/HostConfigurationService.php:47
+msgid "Host name can not be empty"
+msgstr "Le nom d'hôte ne peut être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:105
+#:Centreon/Domain/Engine/EngineService.php:149
+#:Centreon/Domain/Engine/EngineService.php:146
+msgid "The host of service is not defined"
+msgstr "L'hôte du service n'est pas défini"
+
+#:Centreon/Domain/Engine/EngineService.php:172
+msgid "Host of downtime not found"
+msgstr "L'hôte associé à la plage de maintenance n'a pas été trouvé"
+
+#:Centreon/Domain/Engine/EngineService.php:235
+msgid "The host of service %1 is not defined"
+msgstr "L'hôte du service %1 n'est pas défini"
+
+#:Centreon/Domain/Engine/EngineService.php:238
+msgid "Host name of service %1 can not be empty"
+msgstr "Le nom d'hôte du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:241
+msgid "The description of service %1 can not be empty"
+msgstr "La description du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:286
+msgid "Host and service id can not be null at the same time"
+msgstr "L'ID de l'hôte et du service ne peuvent pas être null en même temps"
+
+#:Centreon/Domain/Engine/EngineService.php:289
+msgid "Downtime internal id can not be null"
+msgstr "L'ID de l'hôte et du service ne peuvent pas être null en même temps"
+
+#:Centreon/Domain/Engine/EngineService.php:355
+msgid "Host name cannot be empty"
+msgstr "Le nom d'hôte ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:359
+msgid "Service description cannot be empty"
+msgstr "La description du service ne peut pas être vide"
+
+#:Centreon/Domain/Exception/ExpiredTokenException.php:29
+msgid "Token expired."
+msgstr "Token expiré."
+
+#:Centreon/Domain/Exception/ContactDisabledException.php:29
+msgid "Contact disabled."
+msgstr "Contact désactivé."
+
+#:Centreon/Domain/Gorgone/ActionLog.php:77
+msgid "Token can not empty, null or not defined"
```

Comment:
```suggestion
msgid "Token can not be empty, null or not defined"
```

Reply:
Might broker translate.

Reply:
yeah, the key is already in place, let's not change it